vsphere/requests.yaml

```diff
@@ -5,11 +5,13 @@ releases:
     version: "< 0.26.0"
 - name: "> 29.1.0"
   requests:
-  - name: observability-bundle
-    version: ">= 1.6.2"
   - name: security-bundle
     version: ">= 1.8.2"
 - name: "> 29.0.0"
   requests:
   - name: cert-exporter
     version: ">= 2.9.2"
+- name: "> 28.0.0"
+  requests:
```

Comment:
I think you can't require a changé for v28

Reply:
replaced this with >= 29.1.0